threatstream.json

```diff
@@ -495,6 +495,21 @@
                     "description": "Fetch extra data from Anomali server if available",
                     "data_type": "boolean",
                     "order": 2
+                },
+                "pdns": {
+                    "description": "If enabled, Fetch pdns",
```

Comment:
Should "fetch" be capitalized in these new parameter descriptions?

Reply:
oh good point. What do you think about "If enabled, pdns will also be fetched"?

Reply:
sure

Reply:
super. I will also update `insights` and `external referecnes`